tests/helpers/utils.ts

```diff
@@ -1,11 +1,13 @@
+import debug from 'debug';
 import { type Options, execa } from 'execa';
 import fse from 'fs-extra';
 import fs from 'node:fs/promises';
 import os from 'node:os';
 import path from 'node:path';
 import { fileURLToPath } from 'node:url';
 
-const DEBUG = process.env.DEBUG === 'true';
+const debugLog = debug('addon-blueprint:utils');
```

Comment:
`addon-blueprint:test-utils` maybe? To not suggest users they get debug output when running the bp itself, since this is only effective when running the bp's tests...